app/Http/Controllers/V2/Dashboard/ViewRestorationStrategyController.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace App\Http\Controllers\V2\Dashboard;
+
+use App\Helpers\TerrafundDashboardQueryHelper;
+use App\Http\Controllers\Controller;
+use App\Http\Resources\V2\Dashboard\ViewRestorationStrategyResource;
+use App\Models\V2\Sites\Site;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+
+class ViewRestorationStrategyController extends Controller
+{
+    public function __invoke(Request $request): JsonResponse
+    {
+        $query = $this->buildProjectQuery($request);
+
+        $projectIds = $query->pluck('v2_projects.id')->toArray();
+
+        $restorationStrategy = $this->getRestorationStrategy($projectIds);
+
+        $landUseType = $this->getLandUseType($projectIds);
+
+        $landTenures = $this->getLandTenures($projectIds);
+
+        $result = [
+            'restorationStrategies' => $this->getResultArray($restorationStrategy, 'strategy'),
+            'landUseTypes' => $this->getResultArray($landUseType, 'land_use'),
+            'landTenures' => $this->getResultArray($landTenures, 'land_tenure'),
+        ];
+
+        return new JsonResponse(ViewRestorationStrategyResource::make($result));
+    }
+
+    private function buildProjectQuery(Request $request)
+    {
+
+        $query = TerrafundDashboardQueryHelper::buildQueryFromRequest($request);
+
+        return $query;
+    }
+
+    private function getRestorationStrategy(array $projectIds)
+    {
+        $strategies = ['direct-seeding', 'tree-planting', 'assisted-natural-regeneration'];
```

Comment:
A constant set like this should be a const at the class level. Something like
```
protected const RESTORATION_STRATEGIES = ['direct-seeding', 'tree-planting', 'assisted-natural-regeneration'];
```

Reply:
moved